docs/examples/release/settings.gradle.kts

```diff
@@ -0,0 +1,17 @@
+pluginManagement {
+	repositories {
+		google()
+		mavenCentral()
+		gradlePluginPortal()
```

Comment:
restrict repos to only download known artifacts, e.g. my namespace should only come from mavenCentral/mavenLocal/snapshot repo. and snapshot/mavenLocal should contain no other artifact.